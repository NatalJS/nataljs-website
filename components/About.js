import React from 'react';

const About = () => (
  <section id="about" className="section about">
    <div className="decorative-square" aria-hidden="true" />
    <div className="decorative-circle" aria-hidden="true" />
    <figure className="col-6 col-sm-12 col-md-4 col-lg-3">
      <img className="logo" src="/static/images/logo-colorful.svg" alt="natalJS logo" />
    </figure>
    <div className="col-12 col-sm-12 col-md-8 col-lg-6">
      <h1 className="section-title--alternate">
        O que é a natal JS?
      </h1>
      <article className="article">
        <header>
          <h1 className="article__title">
            Comunidade para discutir javascript em Natal-RN
          </h1>
        </header>
        <p className="article__body">
          Os objetivos são: ajudar os programadores a obter um melhor conhecimento de programação em
          javascript; aproximar programadores locais; organizar encontros, conferências e contribuir
          com projetos open-source relacionados aos estudos.
        </p>
        <footer>
          <a href="https://github.com/NatalJS" className="btn" title="https://github.com/NatalJS">Acesse nosso Github</a>
        </footer>
      </article>
      <div className="about__wave" aria-hidden="true" />
    </div>
  </section>
);

export default About;
