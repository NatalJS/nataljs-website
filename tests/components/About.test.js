import renderer from 'react-test-renderer';
import About from 'components/About';

describe('Component About', () => {
  let Component;

  beforeAll(() => {
    Component = setup(About);
  });

  it('should render component correctly', () => {
    const wrapper = renderer.create(Component());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
