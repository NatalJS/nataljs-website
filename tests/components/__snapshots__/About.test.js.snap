// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component About should render component correctly 1`] = `
<section
  className="section about"
  id="about"
>
  <div
    aria-hidden="true"
    className="decorative-square"
  />
  <div
    aria-hidden="true"
    className="decorative-circle"
  />
  <figure
    className="col-6 col-sm-12 col-md-4 col-lg-3"
  >
    <img
      alt="natalJS logo"
      className="logo"
      src="/static/images/logo-colorful.svg"
    />
  </figure>
  <div
    className="col-12 col-sm-12 col-md-8 col-lg-6"
  >
    <h1
      className="section-title--alternate"
    >
      O que é a natal JS?
    </h1>
    <article
      className="article"
    >
      <header>
        <h1
          className="article__title"
        >
          Comunidade para discutir javascript em Natal-RN
        </h1>
      </header>
      <p
        className="article__body"
      >
        Os objetivos são: ajudar os programadores a obter um melhor conhecimento de programação em javascript; aproximar programadores locais; organizar encontros, conferências e contribuir com projetos open-source relacionados aos estudos.
      </p>
      <footer>
        <a
          className="btn"
          href="https://github.com/NatalJS"
          title="https://github.com/NatalJS"
        >
          Acesse nosso Github
        </a>
      </footer>
    </article>
    <div
      aria-hidden="true"
      className="about__wave"
    />
  </div>
</section>
`;
